src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -15,7 +15,7 @@
 @Table(name = "shopping_cart")
 public class ShoppingCart {
     @Id
-    @Column(name = "shopping_Cart_id")
+    @Column(name = "shopping_cart_id")
     private Long id;
```

Comment:
how is your column in the table named after all? 
`shopping_cart_id` as indicated here, or `id` as indicated in ` @MapsId @JoinColumn(name = "id")`?

consider if you need this `@Column` annotation